activity/obfuscate/README.md

```diff
@@ -0,0 +1,27 @@
+<!--
+title: Obfuscate Activity
+weight: 4618
+-->
+
+# Obfuscate
```

Comment:
I think using just "Obfuscate" as the activity name is too broad.   Maybe "Obfuscate JSON" or "Obfuscate Payload".  The activity directory name/ package should be updated accordingly as well.

Reply:
I changed it to Obfuscate JSON Payload.